### PR TITLE
Link to patch detail

### PR DESCRIPTION
**Pull request by dottorblaster**
This PR adds:
- a new button type called `link` used to make standard effortless programmatic navigation without having to appease a11y guidelines every time
- navigation from SUSE Manager overviews to advisory detail view

### CODE REVIEW

**Comment by dottorblaster** [comment]
@balanza I totally overlooked that, thank you

File: assets/js/common/Button/Button.test.jsx [hunk 1]
```diff
@@ -5,16 +5,64 @@ import '@testing-library/jest-dom';
...
+      expect(screen.getByRole('button')).toHaveTextContent(content);
+    }
+  );
+
+  it.each(sizes)(
+    'should display a button with size %s with its text',
+    (buttonSize) => {
+      const content = faker.vehicle.vehicle();
+      render(
+        <Button size={buttonSize} disabled>
```
Comment (jamie-suse) [inline assets/js/common/Button/Button.test.jsx:61]: Should the button be `disabled` here?

Comment (dottorblaster) [reply assets/js/common/Button/Button.test.jsx:61]: LOL nope my bad, thanks

**Review by janvhs — APPROVED** [review]
Looks great. Thanks for improving the button story as well!

**Review by jamie-suse — APPROVED** [review]

**Pull request merged**